api/src/main/java/marquez/MarquezConfig.java

```diff
@@ -11,15 +11,18 @@
 import io.dropwizard.db.DataSourceFactory;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 import marquez.db.FlywayFactory;
 import marquez.graphql.GraphqlConfig;
+import marquez.jobs.DbRetentionConfig;
 import marquez.service.models.Tag;
 import marquez.tracing.SentryConfig;
 
 /** Configuration for {@code Marquez}. */
 @NoArgsConstructor
 public class MarquezConfig extends Configuration {
   private static final boolean DEFAULT_MIGRATE_ON_STARTUP = true;
+  private static final int DATA_RETENTION_IN_DAYS = 7;
```

Comment:
If we make 7 days the default DB retention behaviour, wouldn't this be considered a breaking change?
Could this lead to users who didn't read the changelog losing data while upgrading to a minor release / setting up new projects?

Reply:
that's a great point.

Reply:
By default, no retention policy is applied to collected metadata in Marquez. I've introduced an [faq](https://github.com/MarquezProject/marquez/blob/feature/db-retention/docs/faq.md) and added _How do I configure a retention policy for metadata?_ section that will better explain how to configure a retention policy.